src/runrms/executor/fm_rms_executor.py

```diff
@@ -37,6 +36,11 @@ class FMRMSExecutor(RMSExecutor):
 
     config: FMRMSConfig
 
+    def __init__(self, config: FMRMSConfig) -> None:
+        super().__init__(config)
```

Comment:
Now initializes everything from the start, so no calling an initializer when `run()` is called